### ISSUE TITLE
Help me out here. What file types are intended here?

### ISSUE THREAD

**Issue by elkym** [sul-dlss/marctable#13]
Your script seems to call for what it terms a .marc file. But I haven't seen that extension used. Is that meant to be a .mrc file? Are they equivalents?

Or is this meant to utilize a .mrk file?

This looks promising.

**Comment by edsu** [comment]
Yes, you will want to find some MARC or MARCXML data. It's kind of an esoteric format, but still used quite a bit in library-land. You can find millions of records available at the Library of Congress if you want to experiment:

https://www.loc.gov/cds/products/MDSConnect-books_all.html
